### PR TITLE
Add iqa 20181227

### PR DESCRIPTION
**Pull request by intelligide**
Specify library name and version:  **iqa/cci.20181227**

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/iqa/all/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,22 @@
+cmake_minimum_required(VERSION 2.8.12)
+project(cmake_wrapper)
+
+include(conanbuildinfo.cmake)
+conan_basic_setup()
+
+if(MSVC AND BUILD_SHARED_LIBS)
+    set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)
```
Comment (prince-chrismc) [inline recipes/iqa/all/CMakeLists.txt:8]: I believe Conan does this automatically

File: recipes/iqa/all/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,22 @@
+cmake_minimum_required(VERSION 2.8.12)
```
Comment (prince-chrismc) [inline recipes/iqa/all/CMakeLists.txt:1]:
```suggestion
cmake_minimum_required(VERSION 3.1)
```

File: recipes/iqa/all/CMakeLists.txt (hunk 1 above)
Comment (intelligide) [reply recipes/iqa/all/CMakeLists.txt:8]: Conan doesn't inject this setting.

Comment (prince-chrismc) [reply recipes/iqa/all/CMakeLists.txt:8]: Even better I just found this gem

https://github.com/conan-io/conan-center-index/pull/2367#discussion_r464125707

Comment (uilianries) [reply recipes/iqa/all/CMakeLists.txt:8]: Indeed Conan doesn't do it. The best approach is asking for the upstream providing it.

Comment (intelligide) [reply recipes/iqa/all/CMakeLists.txt:8]: IQA is unmaintained now.

File: recipes/iqa/all/conanfile.py [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+    def _extract_license(self):
+        content_lines = open(os.path.join(self._source_subfolder, "include", "iqa.h")).readlines()
+        license_content = []
+        for i in range(1, 31):
+            license_content.append(content_lines[i][3:-1])
+        tools.save("LICENSE", "\n".join(license_content))
+
+    def package(self):
+        self.copy(pattern="License.txt", src=self._source_subfolder, dst="licenses")
```
Comment (uilianries) [inline recipes/iqa/all/conanfile.py:69]:
```suggestion
```

**Pull request merged**